core/server/worker/src/main/java/alluxio/worker/page/PagedBlockMetaStore.java

```diff
@@ -232,6 +239,28 @@ private PagedBlockMeta getBlockMetaOfPage(PageId pageId) {
     return blockMeta;
   }
 
+  private boolean containsTempBlockMetaOfPage(PageId pageId) {
```

Comment:
Where is this method called?

Reply:
This is similar with the method `containsBlockMetaOfPage(PageId pageId)`.